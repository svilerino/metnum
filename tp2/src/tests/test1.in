0 0.85 0 tests/test1.txt 0.00001
