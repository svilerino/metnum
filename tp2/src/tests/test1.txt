# Directed graph (each unordered pair of nodes is saved once): 
# Example shown in the Bryan and Leise.
# Nodes: 4 Edges: 8 
# FromNodeId	ToNodeId
1   2
1   3
1   4
2   3
2   4
3   1
4   1
4   3
