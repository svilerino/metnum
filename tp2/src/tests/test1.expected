0.368152
0.141808
0.287962
0.202078
